czech/global.lang.php

```diff
@@ -405,9 +412,10 @@
 $l['custom_mod_tools'] = "Vlastní nástroje";
 
 $l['error_loadlimit'] = "Bylo dosaženo maximální zátěže serveru. Prosím, vraťte se později, až zde nebude takový provoz.";
-$l['error_boardclosed'] = "Toto fórum je právě uzavřeno. Důvody naleznete níže.";
+$l['error_boardclosed'] = "Toto fórum je právě uzavřeno. Důvody naleznete níže.";	
 $l['error_banned'] = "Lituji, ale jste zabanováni. Nemůžete v tomto fóru ani číst, ani psát. Pokud nevíte proč jste ban dostal(a), nebo jej nechcete, kontaktujte administrátora fóra.";
 $l['error_cannot_upload_php_post'] = "Soubor nelze nahrát - je příliš veliký s ohledem na nastavení php (post_max_size). Prosím, klikněte na tlačítko Zpět.";
+$l['error_empty_post_input'] = "There has been an error due to your post data being empty. This could be due to a browser page refresh or direct access to this page. We recommend you press the browser back button and begin again.";
```

Comment:
Preklad

Reply:
Nejsem si úplně jist překladem. Nepodařilo se mi to najít.